artifacts/snip-developer-preview-rhoai.adoc

```diff
@@ -0,0 +1,6 @@
+[IMPORTANT]
+====
+This section describes Developer Preview features in the {openshift-ai-connector-name} plugin. Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to functionality in advance of possible inclusion in a Red Hat product offering. Customers can use these features to test functionality and provide feedback during the development process. Developer Preview features might not have any documentation, are subject to change or removal at any time, and have received limited testing. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
```

Comment:
cool deal this definitely addresses the dev preview discussion we had in the google doc @pabel-rh